### PR TITLE
Translate 3 posts at 2014-10-27 (zh_cn)

### PR DESCRIPTION
**Pull request by debbbbie**
`2014-10-27-ruby-2-0-0-p594-is-released.md`
`2014-10-27-ruby-2-1-4-released.md`
`2014-11-03-tropicalrb-2015-cfp.md`

Screenshots:

ruby-2-0-0-p594-is-released
`2014-10-27-ruby-2-0-0-p594-is-released.md`

ruby-2-1-4-released
`2014-10-27-ruby-2-1-4-released.md`

tropicalrb-2015-cfp
`2014-11-03-tropicalrb-2015-cfp.md`

#1074

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365
@aidewoode @Yunfei1982 @Airmacho @cursorzz @isjia @debbbbie @AndyHelix
@Zane5 @VirgilMing @boyishwei @dandananddada

Pleas help review this.

File: zh_cn/news/_posts/2014-10-27-ruby-2-0-0-p594-is-released.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+lang: zh_cn
+---
+
+很高兴告诉大家 Ruby 2.0.0-p594 发布了。
+
+本次发布包含了 REXML DoS 风险的安全性修正。
+
+* [CVE-2014-8080：XML 扩张的阻断攻击](https://www.ruby-lang.org/zh_cn/news/2014/10/27/rexml-dos-cve-2014-8080/)
+
+本次发布也修改了 ext/openssl 的预设设定。危险的 SSL/TLS 选项预设被关闭了。
```
Comment (aidewoode) [inline zh_cn/news/_posts/2014-10-27-ruby-2-0-0-p594-is-released.md]: I think  the translation style of this sentence is more like Taiwan's style. I think "默认选项" will be better.

**Comment by stomar** [comment]
Please squash into a **single commit** and force push your topic branch. And please use a short commit message, like `Translate 2014-10-27 posts (zh_cn)`

**Comment by JuanitoFatas** [comment]
Thanks for your contributions @debbbbie! Squashed and merged manually as dd7a70d.

**Pull request closed without merging**